test/unit/alphabet/cigar/cigar_op_test.cpp

```diff
@@ -0,0 +1,39 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+#include <gtest/gtest.h>
+
+#include "../alphabet_test_template.hpp"
+#include "../alphabet_constexpr_test_template.hpp"
+#include <seqan3/alphabet/cigar/cigar_op.hpp>
+
+INSTANTIATE_TYPED_TEST_CASE_P(cigar_op, alphabet, cigar_op);
+INSTANTIATE_TYPED_TEST_CASE_P(cigar_op, alphabet_constexpr, cigar_op);
+
+TEST(cigar_op, to_char_assign_char)
+{
+	for (char chr : std::string{"MDISHNPX="})
```

Comment:
but now please indent this line properly... (4 spaces, not 8)
[That's my very last request.]

Reply:
:sweat_smile: Whoops!